lib/xcake/xcode/project.rb

```diff
@@ -4,6 +4,31 @@ module Xcake
   module Xcode
     class Project < Xcodeproj::Project
 
+        # @return [SchemeList] the attributes of
+        #                      the project
+        #
+        def attributes
+          root_object.attributes
+        end
+
+        # @return [SchemeList] the scheme list
+        #
+        def scheme_list
+          @scheme_list ||= SchemeList.new(self)
+        end
+
+        def object_version
+          Xcodeproj::Constants::DEFAULT_OBJECT_VERSION.to_s
+        end
+
+        def recreate_user_schemes(visible = true)
```

Comment:
Unused method argument - `visible`. If it's necessary, use `_` or `_visible` as an argument name to indicate that it won't be used. You can also write as `recreate_user_schemes(*)` if you want the method to accept any arguments but don't care about them.